sql/migrations/relationship-attributes/up.sql

```diff
@@ -43,5 +43,32 @@ CREATE TABLE bookbrainz.relationship_attribute_set__relationship_attribute (
     attribute_id
   )
 );
-	
+
+INSERT INTO bookbrainz.relationship_type ( id, label, description, link_phrase, source_entity_type, target_entity_type, parent_id, child_order, deprecated, reverse_link_phrase ) 
+VALUES 
+( 70, 'Author', 'Indicates the relationship between author and series', 'is part of', 'Author', 'Series', NULL, 0, false, 'contains'),
```

Comment:
How about "Indicates an Author is part of a Series", and the same format for the other relationship types ?